.github/scripts/GitHub.py

```diff
@@ -5,38 +5,80 @@
 #  SPDX-License-Identifier: BSD-2-Clause-Patent
 #
 
+import git
 import logging
 import re
-import requests
 
 from collections import OrderedDict
-from edk2toollib.utility_functions import RunCmd, RunPythonScript
+from edk2toollib.utility_functions import RunPythonScript

```

Comment:
Where is RunPythonScript used?

Reply:
It is called on Line 120 https://github.com/tianocore/edk2/blob/f0c4f302d80b2477f6fd79e1ba2dc3591940f507/.github/scripts/GitHub.py#L120

GitHub Web UI collapses some lines so I use the VS Code GitHub extension.